lib/rules/web/ocm_console/cluster_detail.xyaml

```diff
@@ -302,64 +304,66 @@ check_osd_tab_in_detail_page:
             xpath: //button[@class='pf-c-tabs__button' and contains(., 'Access control')]
         - selector:
             xpath: //button[@class='pf-c-tabs__button' and contains(., 'Add-ons')]
+        - selector:
+            xpath: //button[@class='pf-c-tabs__button' and contains(., 'Networking')]
 check_osd_usage_in_detail_page:
     elements:
         - selector:
-            xpath: //h2[@class='pf-c-title card-title' and text()='Resource usage']
+            xpath: //h2[text()='Resource usage']
         - selector:
-            xpath: //h4[@class='pf-c-title pf-m-xl metrics-chart' and text()='vCPU']
+            xpath: //h4[text()='vCPU']
         - selector:
-            xpath: //h4[@class='pf-c-title pf-m-xl metrics-chart' and text()='Memory']
+            xpath: //h4[text()='Memory']
         - selector:
             xpath: //div[@class='metrics-chart']
 check_osd_details_in_detail_page:
     elements:
         - selector:
-            xpath: //h2[@class='pf-c-title card-title' and text()='Details']
+            xpath: //h2[text()='Details']
         - selector:
-            xpath: //dl[@class='cluster-details-item left']/dt[text()='Cluster ID']
+            xpath: //dl[@class='cluster-details-item']/dt[text()='Cluster ID']
         - selector:
-            xpath: //dl[@class='cluster-details-item left']/dt[text()='Type']
+            xpath: //dl[@class='cluster-details-item']/dt[text()='Type']
         - selector:
-            xpath: //dl[@class='cluster-details-item left']/dd[text()='OSD']
+            xpath: //dl[@class='cluster-details-item']/dd/span[text()='OSD']
```

Comment:
Better not to use the class to locate. It's better to use //dl[1]/dt[]
And should package the public part of OCP and OSD to an action

Reply:
Updated